NotationV0/task.json

```diff
@@ -0,0 +1,198 @@
+            "type": "string",
+            "label": "Artifact references",
+            "defaultValue": "",
+            "required": false,
+            "visibleRule": "command = verify || command = sign",
+            "helpMarkDown": "Container artifact references for signing. If it was not specified, use the artifact reference from previous Docker push task. Example: <registry name>/<repository name>@<digest> If multiple, separate by comma.",
+            "groupName": "commandConfig"
+        },
+        {
+            "name": "signatureFormat",
+            "type": "pickList",
+            "label": "Signature Format",
+            "defaultValue": "cose",
+            "required": false,
+            "options": {
+                "cose": "cose",
+                "jws": "jws"
+            },
+            "visibleRule": "command = sign",
+            "helpMarkDown": "Signature envelope format, options: \"jws\", \"cose\" (default \"jws\")",
+            "groupName": "advancedConfig"
+        },
+        {
+            "name": "allowReferrersAPI",
+            "type": "boolean",
+            "label": "[Experimental] Allow Referrers API",
+            "defaultValue": false,
+            "required": false,
+            "visibleRule": "command = sign || command = verify",
+            "helpMarkDown": "Use the Referrers API to sign signatures, if not supported (returns 404), fallback to the Referrers tag schema",
+            "groupName": "advancedConfig"
+        },
+        {
+            "name": "plugin",
+            "type": "pickList",
+            "label": "Plugin",
+            "defaultValue": "azureKeyVault",
+            "required": true,
+            "options": {
+                "azureKeyVault": "Azure Key Vault Plugin"
+            },
+            "groupName": "pluginConfig"
+        },
+        {
+            "name": "azurekvServiceConection",
+            "type": "connectedService:AzureRM",
+            "label": "Azure Key Vault service connection",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Select the Azure subscription for the key vault if prefer to use service connection for authentication.",
+            "visibleRule": "plugin = azureKeyVault",
+            "groupName": "pluginConfig"
+        },
+        {
+            "name": "keyid",
+            "type": "string",
+            "label": "Key ID",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "The Key ID is the key or certificate identifier for Azure Key Vault.",
+            "visibleRule": "plugin = azureKeyVault",
+            "groupName": "pluginConfig"
+        },
+        {
+            "name": "caCertBundle",
+            "type": "string",
+            "label": "Certificate Bundle File Path",
+            "required": false,
+            "defaultValue": "",
+            "visibleRule": "plugin = azureKeyVault",
+            "groupName": "pluginConfig"
```

Comment:
Since you had the concern of user do not know what a `cert bundle` is, shall we add a more detailed description or `helpMarkDown` here?
For example, `a file with root and all intermediate certificates following the order of the certificate chain`, something like this could help.

Reply:
Updated.